projects/observability/src/shared/components/user-specified-time-range-selector/user-specified-time-range-selector.component.ts

```diff
@@ -0,0 +1,37 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
```

Comment:
This component is just a wrapped around `TimeRangeComponent` and decides whether or not to save a time range to storage when the user selects it. Criteria for saving to storage are having a only a single path in the url and having the `defaultTimeRange` set in the `data` property of the activated route.